sui/operators.js

```diff
@@ -0,0 +1,301 @@
+    });
+
+    tx.moveCall({
+        target: `${gasServiceConfig.address}::gas_service::collect_gas`,
+        arguments: [tx.object(gasServiceConfig.objects.GasService), cap, tx.pure.address(receiver), tx.pure.u64(amount)],
+    });
+
+    tx.moveCall({
+        target: `${operatorsConfig.address}::operators::restore_cap`,
+        arguments: [
+            tx.object(operatorId),
+            tx.object(operatorCapId),
+            tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes()),
+            cap,
+            loanedCap,
+        ],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    const receipt = await broadcast(client, keypair, tx);
+
+    printInfo('Gas collected', receipt.digest);
+}
+
+async function refund(keypair, client, config, chain, args, options) {
+    const [messageId] = args;
+    const amount = options.amount;
+    const receiver = options.receiver || keypair.toSuiAddress();
+
+    const gasServiceConfig = config.sui.contracts.GasService;
+    const operatorsConfig = chain;
+
+    if (!gasServiceConfig) {
+        throw new Error('Gas service package not found.');
+    }
+
+    if (!operatorsConfig) {
+        throw new Error('Operators package not found.');
+    }
+
+    const operatorId = operatorsConfig.objects.Operators;
+    const gasCollectorCapId = await getGasCollectorCapId(client, gasServiceConfig, operatorsConfig);
+    const operatorCapId = await findOwnedObjectId(client, keypair.toSuiAddress(), `${operatorsConfig.address}::operators::OperatorCap`);
+
+    const tx = new Transaction();
+
+    const [cap, loanedCap] = tx.moveCall({
+        target: `${operatorsConfig.address}::operators::loan_cap`,
+        arguments: [tx.object(operatorId), tx.object(operatorCapId), tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes())],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    tx.moveCall({
+        target: `${gasServiceConfig.address}::gas_service::refund`,
+        arguments: [
+            tx.object(gasServiceConfig.objects.GasService),
+            cap,
+            tx.pure.string(messageId),
+            tx.pure.address(receiver),
+            tx.pure.u64(amount),
+        ],
+    });
+
+    tx.moveCall({
+        target: `${operatorsConfig.address}::operators::restore_cap`,
+        arguments: [
+            tx.object(operatorId),
+            tx.object(operatorCapId),
+            tx.pure(bcs.Address.serialize(gasCollectorCapId).toBytes()),
+            cap,
+            loanedCap,
+        ],
+        typeArguments: [`${gasServiceConfig.address}::gas_service::GasCollectorCap`],
+    });
+
+    const receipt = await broadcast(client, keypair, tx);
+
+    printInfo('Gas refunded', receipt.digest);
+}
+
+async function storeCap(keypair, client, config, chain, args, options) {
+    const [capId] = args;
+    const gasCollectorCapConfig = config.sui.contracts.GasService;
```

Comment:
```suggestion
    const gasServiceConfig = config.sui.contracts.GasService;
```
This can be passed in as an arg as well